Cypress fails on CI due to deprecated har-validator dependency

See for example: https://app.travis-ci.com/github/opfab/operatorfabric-core/builds/242396111

Error message

> > cypress@9.1.0 postinstall /home/travis/build/opfab/operatorfabric-core/src/test/cypress/node_modules/cypress
> > node index.js --exec install
> internal/modules/cjs/loader.js:883
>   throw err;
>   ^
> Error: Cannot find module 'har-validator'
> Require stack:
> - /home/travis/build/opfab/operatorfabric-core/src/test/cypress/node_modules/@cypress/request/lib/har.js

This is due to https://github.com/cypress-io/cypress/issues/19102

--- a/src/test/cypress/package.json
+++ b/src/test/cypress/package.json
@@ -10,6 +10,7 @@
   "license": "MPL-2.0",
   "devDependencies": {
     "cypress": "9.1.0",
-    "cypress-terminal-report": "3.4.0"
+    "cypress-terminal-report": "3.4.0",
+    "har-validator": "5.1.5"
   }
 }

